### PR TITLE
413 404 page

### LINKED ISSUE #413: [FRONTEND] Create and implement 404 page
**Is your feature request related to a problem? Please describe.**
will need to do something like:
```tsx
    <Route path="*">
            <NoMatch />
          </Route>
```

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #413

### CODE REVIEW

**Comment by zlrkw11** [comment]
image
- set up route in router
- auto directs to the last page clicked

**Comment by bcho892** [comment]
image

can we center this vertically? in this case its fine to use screen height

File: client/src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// import React from "react"
```
Comment (bcho892) [inline client/src/pages/404.tsx:1]: do not comment out unused code, need to delete it

File: client/src/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// import { useNavigate } from "react-router-dom"
+import Wind from "assets/logos/Wind.png"
+import Button from "components/generic/FigmaButtons/FigmaButton"
+
+const NoMatch = () => {
+  const handleBack = () => {
+    window.history.go(-1)
```
Comment (bcho892) [inline client/src/pages/404.tsx:8]: we should navigate back to home page, also should use `react-router` functions for this instead (prefer to redirect to home)

File: client/src/pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  return (
+    <div className="flex flex-col items-center justify-center gap-4">
+      <img src={Wind}></img>
+
+      <h2>Error 404</h2>
+      <h4>Sorry! Page not found</h4>
+      <div className="pt-[2.5%]">
```
Comment (bcho892) [inline client/src/pages/404.tsx:17]: nit: percentage margins not preferred

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: client/src/pages/404.tsx (hunk 3 above)
Comment (zlrkw11) [reply client/src/pages/404.tsx:17]: changed to px

**Comment by zlrkw11** [comment]
image
- content centered vertically
- react-router implemented
- comments deleted

**Comment by zlrkw11** [comment]
image
- mobile viewport

**Comment by bcho892** [comment]
The image quality is very bad, can we use an svg instead?

**Comment by zlrkw11** [comment]
image
- new svg imported

**Review by bcho892 — APPROVED** [review]

**Pull request merged**